ovh/ovh.go

```diff
@@ -321,6 +304,12 @@ func (c *Client) CallAPI(method, path string, reqBody, resType interface{}, need
 
 	// Send the request with requested timeout
 	c.Client.Timeout = c.Timeout
+
+	return req, nil
+}
+
+// DoRequest and unmarshall response into resType
+func (c *Client) DoRequest(req *http.Request, resType interface{}) error {
```

Comment:
Why not just `Do` ?
This should almost be like a sentence: c.Do(req) == `client do request` and not `client do  request request`